src/tellor_disputables/data.py

```diff
@@ -97,17 +98,20 @@ async def log_loop(web3: Web3, addr: str) -> list[tuple[int, Any]]:
     return unique_events_lis
 
 
-def is_disputable(reported_val: float, query_id: str, conf_threshold: float) -> Optional[bool]:
+async def is_disputable(reported_val: float, query_id: str, conf_threshold: float) -> Optional[bool]:
     """Check if the reported value is disputable."""
     if query_id not in DATAFEED_LOOKUP:
         print(f"new report for unsupported query ID: {query_id}")
         return None
 
-    current_feed = DATAFEED_LOOKUP[query_id]
-    trusted_val = asyncio.run(current_feed.source.fetch_new_datapoint())[0]
-    percent_diff = (reported_val - trusted_val) / trusted_val
-
-    return bool(abs(percent_diff) > conf_threshold)
+    current_feed: DataFeed[Any] = DATAFEED_LOOKUP[query_id]
+    trusted_val: Optional[float] = (await current_feed.source.fetch_new_datapoint())[0]
```

Comment:
Extra pair of parentheses here

Reply:
required, as otherwise it indexes the coroutine itself